sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -903,6 +904,11 @@ def create_trigger_driver(windowing,
   # sorted order.
   if windowing.is_default() and is_batch:
     driver = DefaultGlobalBatchTriggerDriver()
+  elif (windowing.windowfn == GlobalWindows()
```

Comment:
Would this branch also apply to streaming?
If so, it might be a good idea to remove the word `Batch` from the driver's name.

Reply:
Done.